gateway/handle.go

```diff
@@ -759,71 +769,52 @@ func (gw *Handle) extractJobsFromInternalBatchPayload(
 			continue
 		}
 
-		userID := buildUserID("", anonIDFromReq, userIDFromReq)
-		receivedAt, _ := toSet["receivedAt"].(string)
-		if receivedAt == "" {
-			receivedAt = time.Now().Format(misc.RFC3339Milli)
+		jobsDBParams := params{
+			MessageID:       msg.Properties.MessageID,
+			SourceID:        msg.Properties.SourceID,
+			SourceJobRunID:  msg.Properties.SourceJobRunID,
+			SourceTaskRunID: msg.Properties.SourceTaskRunID,
+			UserID:          msg.Properties.UserID,
+			TraceParent:     msg.Properties.TraceID,
+		}
+
+		marshalledParams, err := json.Marshal(jobsDBParams)
+		if err != nil {
+			gw.logger.Errorn(
+				"[Gateway] Failed to marshal parameters map. Parameters: %+v",
+				logger.NewField("params", jobsDBParams),
+				obskit.Error(err),
+			)
```

Comment:
We don't need `Parameters: %+v`.
```suggestion
			gw.logger.Errorn("[Gateway] Failed to marshal parameters map",
				logger.NewField("params", jobsDBParams),
				obskit.Error(err),
			)
```